enos/modules/docker_worker/worker-config-bsr-downstream.hcl

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+disable_mlock = true
+
+telemetry {
+  prometheus_retention_time = "24h"
+  disable_hostname          = true
+}
+
+listener "tcp" {
+  address     = "0.0.0.0:${port}"
+  purpose     = "proxy"
+  tls_disable = true
+}
+
+worker {
+  name = "${worker_name}"
+  initial_upstreams = ["${initial_upstream}"]
+
+  tags {
+    type = ${type_tags},
+  }
+
+  recording_storage_path = "${recording_storage_path}"
+}
+
+kms "aead" {
+  purpose   = "worker-auth"
+  aead_type = "aes-gcm"
+  key       = "8fZBjCUfN0TzjEGLQldGY4+iE9AkOvCfjh7+p0GtRBQ="
```

Comment:
It was important that this matched the upstream worker right? Would it be worth adding a comment here and in the other place?